src/app/examples/floating-action-button-example/floating-action-button-example.component.scss

```diff
@@ -0,0 +1,4 @@
+kirby-floating-action-button {
+    margin: 10px;
```

Comment:
Can this be value be obtained from one of the kirby-sizes?

Reply:
Given that this style is only used in the example, I don't think there should be a standard of how separate floating action buttons should be from each other in a possible application.